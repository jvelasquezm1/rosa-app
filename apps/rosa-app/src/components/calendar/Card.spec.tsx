import { Card } from '@material-ui/core';
import renderer from 'react-test-renderer';

describe('Card component', () => {
    it('should render successfully', () => {
        const component = renderer.create(<Card />)
        expect(component.toJSON()).toMatchSnapshot()
    });
})