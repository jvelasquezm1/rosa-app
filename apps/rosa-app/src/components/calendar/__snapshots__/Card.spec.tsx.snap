// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render successfully 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
/>
`;
